### PR TITLE
Added the lux card to the Absence Request index detail view

### PR DESCRIPTION
**Pull request by carolyncole**
Added an AbsenceRequest decorator to clean up data being displayed on the card

continues #184

### CODE REVIEW

File: spec/decorators/absence_request_decorator_spec.rb [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  describe "#formatted_end_date" do
+    let(:absence_request) { FactoryBot.create(:absence_request, end_date: Time.zone.parse("2019-07-04 12:12")) }
+    it "returns a formated start date" do
+      expect(absence_request_decorator.formatted_end_date).to eq "Jul 4, 2019"
+    end
+  end
+
+  describe "#status_icon" do
+    let(:absence_request) { FactoryBot.create(:absence_request) }
+    it "returns a formated start date" do
```
Comment (hackmastera) [inline spec/decorators/absence_request_decorator_spec.rb]: this spec title needs to be updated

File: spec/decorators/absence_request_decorator_spec.rb [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  describe "#absence_type_icon" do
+    context "when absence_type is vacation" do
+      it "returns vacation icon" do
+        expect(absence_request_decorator.absence_type_icon).to eq "lux-icon-vacation"
+      end
+    end
+    context "when absence_type is sick" do
+      let(:absence_request) { FactoryBot.create(:absence_request, absence_type: :sick) }
+      it "returns sick icon" do
+        expect(absence_request_decorator.absence_type_icon).to eq "lux-icon-hospital"
```
Comment (hackmastera) [inline spec/decorators/absence_request_decorator_spec.rb]: do we want more of these tests? For title I wrote them all; not sure if we want to do that here, also

Comment (carolyncole) [reply spec/decorators/absence_request_decorator_spec.rb]: Sure we can have more tests.

**Comment by carolyncole** [comment]
@hackmastera Added more tests and updated the comment.

File: spec/decorators/absence_request_decorator_spec.rb [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    end
+    context "when absence_type is work_from_home" do
+      let(:absence_request) { FactoryBot.create(:absence_request, absence_type: :work_from_home) }
+      it "returns work_from_home icon" do
+        expect(absence_request_decorator.absence_type_icon).to eq "lux-icon-clock"
+      end
+    end
+    context "when absence_type is consulting" do
+      let(:absence_request) { FactoryBot.create(:absence_request, absence_type: :consulting) }
+      it "returns sick consulting" do
```
Comment (hackmastera) [inline spec/decorators/absence_request_decorator_spec.rb]: one more spec title that needs updated!

**Pull request merged**